screens/buddies.js

```diff
@@ -25,6 +25,7 @@ export default function Buddies({ navigation }) {
 
     return (
         <View style={dyStyles.buddyDisplayContainer}>
+            <Text style={dyStyles.title_text}>Tap your buddy for more info!</Text>
```

Comment:
How about "Select on a Buddy for more info!"? Maybe this is just me being nitpicky hahaha

Reply:
I like what is there. I think having it explicitly say "Tap" is good. It shouldn't allow for any confusion.